src/config/auth.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
-  'googleAuth': {
-    'clientID': '725194704366-4nlruh4oc7g2lrtfok18qqacnq7h2o2m.apps.googleusercontent.com',
-    'clientSecret': 'xjJrXQs2KRT3NBqwXPBue5sy',
-    'callbackURL': 'http://localhost:8000/api/auth/google/callback'
+  googleAuth: {
```

Comment:
We store all our configuration in the `chill.yml` file. No need additional config files.